README.md

```diff
@@ -66,14 +66,14 @@ The following Helm values are available:
 Linkerd-smi installation:
 
 ```console
-helm repo add linderd-smi https://linkerd.github.io/linkerd-smi
+helm repo add linkerd-smi https://linkerd.github.io/linkerd-smi
 helm install linkerd-smi -n linkerd-smi --create-namespace linkerd-smi/linkerd-smi
 ```
 
 Linkerd-failover installation:
 
 ```console
-helm install linkerd-failover -n linkerd-failover --create-namespace --devel linkerd/linkerd-failover
+helm install linkerd-failover -n linkerd-failover --create-namespace --devel charts/linkerd-failover
```

Comment:
Is this the correct change?  Changing this was necessary to get it to work for me.  @alpeb

Reply:
Your change is fine if you want to install from the source code. I think we need to have another section on how to pull the published helm repo (should be the same as linkerd's) and install using that repo.